userspace/libscap/libscap.pc.in

```diff
@@ -1,10 +1,11 @@
-prefix=${pcfiledir}/../..
+prefix=@CMAKE_INSTALL_PREFIX@
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
 Name: libscap
 Description: lib for System CAPture
 Version: @FALCOSECURITY_LIBS_VERSION@
 
-Libs: -L${libdir} @LIBSCAP_LINK_LIBDIRS_FLAGS@ @LIBSCAP_LINK_LIBRARIES_FLAGS@
-Cflags: -I${includedir}/@LIBS_PACKAGE_NAME@/libscap
+Requires: zlib
+Libs: -L${libdir} -L{libdir}/@LIBS_PACKAGE_NAME@/libscap @LIBSCAP_LINK_LIBDIRS_FLAGS@ @LIBSCAP_LINK_LIBRARIES_FLAGS@
+Cflags: -I${includedir}/@LIBS_PACKAGE_NAME@/libscap -I${includedir}/@LIBS_PACKAGE_NAME@/driver -I${includedir}/@LIBS_PACKAGE_NAME@
```

Comment:
We should probably add `-I{$UTHASH_INCLUDE}` here in order to be more strictly compliant with the pkg-config specification.

It also looks like the ".../libscap" and ".../driver" entries aren't strictly needed as of 6f8cb4707. I suppose it's a question of how convenient we want to make things for people who might include "scap.h" instead of "userspace/scap.h".

Reply:
I've now added -I@UTHASH_INCLUDE@ to libscap.pc.in, thanks for pointing that.  About whether we should allow the includes to work without the libscap and libsinsp prefixes, I'll leave the call to you, but note that at least some test files still do:

```

userspace/libsinsp/test/classes/sinsp.cpp:21:#include <scap.h>
userspace/libsinsp/test/plugins/plugin_source.cpp:24:#include <scap.h>
userspace/libsinsp/test/plugins/syscall_async.cpp:26:#include <scap.h>
userspace/libsinsp/test/plugins/syscall_source.cpp:26:#include <scap.h>
```

as of current master.